src/analysis/normed_space/basic.lean

```diff
@@ -169,6 +169,10 @@ lemma tendsto_iff_norm_tendsto_zero {f : ι → β} {a : filter ι} {b : β} :
   tendsto f a (nhds b) ↔ tendsto (λ e, ∥ f e - b ∥) a (nhds 0) :=
 by rw tendsto_iff_dist_tendsto_zero ; simp only [(dist_eq_norm _ _).symm]
 
+lemma tendsto_at_top_iff_norm_tendsto_zero [nonempty ι] [lattice.semilattice_sup ι] {u : ι → α} {a : α} :
```

Comment:
Why is this stated only for `at_top`? Isn't it true for any filter (with the same proof)?

Reply:
I've removed the lemma.